README.md

```diff
@@ -242,6 +242,10 @@ Language scale factors were derived from Mayes Consulting, LLC web site
 http://softwareestimator.com/IndustryData2.htm.
 [](1}}})
 <a name="apt-get"></a> []({{{1)
+## Run through docker
+```shell
+docker run --rm -w /cloc -v $PWD:/cloc cloc
```

Comment:
I could suggest to rename the mounted directory in this example to avoid confusion between the name of the directory and application name: 

```suggestion
docker run --rm -w /workdir -v $PWD:/workdir cloc
```